src/UI/Components/Diagram/Canvas.tsx

```diff
@@ -156,11 +156,15 @@ export const Canvas: React.FC<Props> = ({ editable }) => {
       <DictModal />
       <CanvasWrapper id="canvas-wrapper" data-testid="Composer-Container">
         <LeftSidebarContainer
-          className="left_sidebar"
+          className={`left_sidebar ${!editable && "view_mode"}`}
```

Comment:
🤔 why do we need both assertions? Doesn't !editable imply that you are in view-mode?

Reply:
It's not assertion, we append class there